RPIPs/RPIP-49.md

```diff
@@ -100,32 +100,27 @@ This RPIP introduces a delay after protocol upgrades have been confirmed but pri
 The tokenomics rework package will likely be split between two protocol upgrades: Saturn 1 and Saturn 2.
 
 ### Saturn 1
-
-* [RPIP-43: Megapools](RPIP-43.md)
-  * Including ETH-only validators.
-* [RPIP-42: Bond curves](RPIP-42.md)
-  * Framework
-  * 4ETH minimum bond
-* [RPIP-59: Deposit Mechanics](RPIP-59.md)
-  * Standard and express queues.
-* [RPIP-46: Universal Adjustable Revenue Split](RPIP-46.md)
-  * Splits between rETH, node operators, and vote-eligible staked RPL staked in megapools
-  * RPL issuance rewards no longer have a minimum stake required
-* RPL Value Capture - Increased share to voting Node Operators.
+* [RPIP-42: Bond curves](RPIP-42.md) - Partial Inclusion
+  * 4 ETH minimum bond
+* [RPIP-43: Megapools](RPIP-43.md) (includes ETH-only validators)
```

Comment:
I kinda liked the bullet point better :sweat_smile:, but nbd either way. I like "Includes" better than "including" though

Reply:
Eh, I think I will leave it.